src/edu/csus/ecs/pc2/clics/API202306/ContestService.java

```diff
@@ -389,9 +388,15 @@ private Response HandleContestCountdownPauseTime(SecurityContext sc, String cont
         // MIN_VALUE is returned on format error
         if(pauseTime != Long.MIN_VALUE) {
             // want to stop the countdown with this many milliseconds left
-            // TODO: tell PC2 to stop countdown when clock is 'pauseTime' ms away from start
-            controller.getLog().log(Log.WARNING, LOG_PREFIX + contestId + ": countdown_pause_time not implemented");
-            return Response.status(Status.NOT_MODIFIED).entity("Unable to set countdown_pause_time to " + pauseTime + " ms").build();
+            // tell PC2 to stop countdown when clock is 'pauseTime' ms away from start
+            boolean success = setScheduledStart(null, countdownPauseTime);
+            if (success) {
+                controller.getLog().log(Log.INFO, LOG_PREFIX + contestId + ": countdown paused (set contest start time to null)");
+                return Response.ok().entity("Contest countdown paused - start time updated to \"null\" (no scheduled start)").build();
```

Comment:
Analogous to the above comment: I think it would helpful if the Response message included the "pause time" -- i.e. if the message was (something like) `Contest countdown paused at 00:05:00 and start time updated to "null" (no scheduled start)`.

Reply:
Fixed in next push